packages/recorder/src/callLog.tsx

```diff
@@ -54,7 +54,7 @@ export const CallLogView: React.FC<CallLogProps> = ({
             { message.trim() }
           </div>;
         })}
-        { !!callLog.error && <div className='call-log-message error' hidden={!isExpanded}>{ callLog.error.error?.message }</div> }
+        { !!callLog.error && <div className='call-log-message error' hidden={!isExpanded}>{ callLog.error }</div> }
```

Comment:
Stray change?

Reply:
It is not stray, I needed to simplify the interface to remove channel dependencies.